snuba/processor.py

```diff
@@ -70,6 +72,52 @@ class InvalidMessageVersion(SerializableException):
     pass
 
 
+class ReplacementType(str, Enum):
```

Comment:
These were moved from `events_processor_base`.

Reply:
These seem a little too specific to be in such a high level file. Wouldn't ReplacementType(s) live with the replacement processor?

Reply:
Potentially, but right now this type is imported in `snuba/datasets` as well as `snuba/replacers` and I think organizing the replacement code is out of scope for this PR.